abstracts/_functions.scss

```diff
@@ -0,0 +1,8 @@
+$color-green: #679281;
```

Comment:
Hi Tim, All looks good, the only thing I would probably change is moving your color variables into _variables.scss rather than _functions.scss.